flang/include/flang/Lower/CallInterface.h

```diff
@@ -170,7 +170,7 @@ class CallInterface {
     /// Is the argument INTENT(OUT)
     bool isIntentOut() const;
     /// Does the argument have the CONTIGUOUS attribute or have explicit shape?
-    bool mustBeMadeContiguous() const;
+    bool mustBeMadeContiguous(const bool argHasTriplet = false) const;
```

Comment:
`const` is never needed in a prototype argument that isn't a reference or a pointer